app/src/main/java/org/p2p/wallet/striga/signup/presetpicker/SelectItemProvider.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.striga.signup.presetpicker
+
+import androidx.recyclerview.widget.RecyclerView
+
+interface SelectItemProvider {
+    fun provideItemsName(): SelectItemsStrings
+
+    fun provideItemDecorations(): List<RecyclerView.ItemDecoration> = emptyList()
```

Comment:
item decorations to use when rendering the universal list, group / add paddings and etc as you like